### PR TITLE
Update `servo-tidy` to 0.3.0

### PR DESCRIPTION
**Pull request by UK992**
r? @Wafflespeanut 

It's ready to publishing to PyPI.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/15731)

### CODE REVIEW

File: python/tidy/HISTORY.rst [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
 
-0.1.0 (2016-08-09)
+0.3.0 (2017-02-24)
+++++++++++++++++++
+
+- Added checking for blocked packages
+- Added checking for alphabetical order for JSON keys and ``#![feature(...)]``
+- Improved duplicated packages checking
+- Improved ordering ``use`` statements
+- And many others improvements
```
Comment (Wafflespeanut) [inline python/tidy/HISTORY.rst]: Nit: `other`

And, you missed mine - [A lint runner for running custom lints](https://github.com/servo/servo/pull/14166) 😄

**Comment by Wafflespeanut** [comment]
r=me after addressing that :)

@bors-servo delegate+

File: python/tidy/HISTORY.rst (hunk 1 above)
Comment (UK992) [inline python/tidy/HISTORY.rst]: done!

**Comment by Wafflespeanut** [comment]
@bors-servo r+

(cc @edunham)

**Pull request merged**